cmd/testclient/client.go

```diff
@@ -320,6 +323,35 @@ func main() {
 			os.Exit(1)
 		}
 		sugar.Infow("DeleteCluster method", "response", v)
+	case "DeleteAllClustersInRegion":
+		v, err := client.GetClusters(context.Background(), getClustersReq)
+		if err != nil {
+			sugar.Errorw("error getting clusters", "account", getClustersReq.AccountName, "provider", getClustersReq.Provider, "region", getClustersReq.Region, "error", err)
+			os.Exit(1)
+		}
+		clusters := make([]string, 0)
+		for _, cluster := range v.Clusters {
+			clusters = append(clusters, cluster.Name)
+		}
+		logger.Sugar().Infow("deleting following clusters", "clusters", clusters)
+
+		for _, cluster := range clusters {
+			sugar.Infow("deleting cluster", "cluster", cluster)
+			req := &proto.ClusterDeleteRequest{
+				Provider:    provider,
+				Region:      region,
+				AccountName: accountName,
+				ClusterName: cluster,
+				ForceDelete: true,
+			}
+			v, err := client.DeleteCluster(context.Background(), req)
+			if err != nil && err.Error() != "" {
+				sugar.Errorw("error deleting cluster", "cluster", cluster, "error", err)
```

Comment:
may be add continue when fails.

lets simplify this `if err != nil && err.Error() != ""` to `if err != nil`.

Reply:
fixed